templates/repo/settings/webhook/settings.tmpl

```diff
@@ -354,3 +364,20 @@
 		>{{ctx.Locale.Tr "repo.settings.delete_webhook"}}</a>
 	{{end}}
 </div>
+
+<script>
+	(function () {
+		const section = document.querySelector('.events.fields.ui.grid');
+		if (!section) return;
+
+		const all = section.querySelectorAll('input[type="checkbox"]');
+
+		document.getElementById('event-select-all')?.addEventListener('click', () => {
+			all.forEach(i => { i.checked = true; });
+		});
+
+		document.getElementById('event-deselect-all')?.addEventListener('click', () => {
+			all.forEach(i => { i.checked = false; });
+		});
+	})();
+</script>
```

Comment:
1. We need to move this code to `web_src/js/features/repo-settings.ts`. We avoid inline scripts because they are problematic with CSP and also the linter does not lint them.
2. Use `for-of` loops instead of `forEach`, but the linter will also tell you that 😉.